src/webgl/p5.Geometry.js

```diff
@@ -142,6 +142,24 @@ p5.Geometry = class Geometry {
     this.vertexColors = [];
     return this;
   }
+  flipU() {
```

Comment:
It would be good to add some doc comments for these methods too so we can show some examples of their usage.

Reply:
Should it be like this?
```javascript
/**
 * Flip the U texture coordinates of the model.
 * @method flipU
 * @for p5.Geometry
 *
 * @returns {p5.Geometry} Returns the modified p5.Geometry object.
 *
 * @example
 * <div>
 * <code>
 * let model;
 * 
 * function preload() {
 *   model = loadModel('path/to/your/model.obj', modelLoaded);
 * }
 * 
 * function modelLoaded() {
 *   model.flipU();
 *   
 *   createCanvas(400, 400, WEBGL);
 *   background(200);
 *   
 *   // Original model (no texture coordinate flipping)
 *   fill('orange');
 *   translate(-100, 0, 0);
 *   model(model);
 *   
 *   // Flipped U texture coordinates model
 *   fill('blue');
 *   translate(200, 0, 0);
 *   model(model);
 * }
 * </code>
 * </div>
 */
```

Reply:
Right, like that! Maybe instead of doing everything in a success callback, we can omit the callback, and just rename `modelLoaded` to `setup` (since it will wait for preload to finish before running it anyway.)